### DISCUSSION TITLE
Please tell me where I can see the trends of I following people.

### DISCUSSION THREAD

**Discussion by bigbigbo** [community/community#66563] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Please tell me where I can see the trends of I following people.

I try any filters, but the result is not I want.

**Comment by ShahbazShaddy** [comment]
GitHub does not have a built-in feature to show trends or activity specifically for the people you are following. The "Trending" section on GitHub primarily displays popular repositories and projects across the platform, not personalized trends related to the activities of users you follow.

If you're interested in keeping track of the activities of specific users, you might need to use third-party tools or services that offer this functionality. Some developers and organizations have created external tools or browser extensions that provide more personalized insights into GitHub activity.

However, please note that it's essential to exercise caution when using third-party tools or extensions and ensure they are from reputable sources to protect your account security.

**Reply by bigbigbo** [reply]
github redesigned the home page, before that, I can see the activities of people I following, like someone `star` or `fork` a repository.

**Comment by nbelyh** [comment, marked as answer]
You cannot anymore, they have changed the feed. Now you should follow what GitHub wants you to follow.

**Reply by bigbigbo** [reply]
Ok, I still like the previous design.
